Phonebook.Frontend/src/styles.scss

```diff
@@ -136,6 +142,11 @@ app-root {
   justify-content: center;
 }
 
+.pb-orthogonal-center-element {
```

Comment:
`.pb-center-text` should also work 🤔

Reply:
If not you can rename it to `.pb-center-element-orthogonal` please

Reply:
> `.pb-center-text` should also work 🤔

not it doesn't this class has flex-flow: column, that's totally different